config/sync/core.entity_view_display.node.stanford_event.search_indexing.yml

```diff
@@ -190,6 +195,7 @@ hidden:
   links: true
   search_api_excerpt: true
   stanford_intranet__access: true
+  su_event_contact_info: true
```

Comment:
We probably want it to be indexed. This is the display mode that does that for us. Remember to hide the label though.

Reply:
Fixed.